src/Interpreters/AsynchronousMetrics.cpp

```diff
@@ -713,9 +725,10 @@ void AsynchronousMetrics::update(TimePoint update_time)
             /// Log only if difference is high. This is for convenience. The threshold is arbitrary.
             if (difference >= 1048576 || difference <= -1048576)
                 LOG_TRACE(log,
-                    "MemoryTracking: was {}, peak {}, will set to {} (RSS), difference: {}",
+                    "MemoryTracking: was {}, peak {}, allocator {}, will set to {} (RSS), difference: {}",
```

Comment:
How about `allocator fragmentation` ?

Reply:
I'm afraid that "allocator fragmentation" wouldn't tell anything even for CH engineers )
for "free memory in arenas" google provides pretty clear explanation

Reply:
I'm ok with any option. Though I though that `fragmentation` for allocators should be pretty generic, since it always has it.

P.S. google also shows enough information for `allocator fragmentation` request.

Reply:
@nickitat so, do you want to go with `free memory in arenas` ?